tests/index.test.js

```diff
@@ -59,7 +59,15 @@ icons.forEach(icon => {
     }
   });
 
-  test(`${icon.title} can be found by it's slug`, () => {
+  test(`${icon.title} can be 'Get' by it's slug`, () => {
```

Comment:
Suggested change here for readbility. Other than that, everything looks good to me.
```suggestion
  test(`'Get' ${icon.title} by its slug`, () => {
```